core/src/sumeragi/mod.rs

```diff
@@ -55,13 +55,13 @@ struct LastUpdateMetricsData {
 #[derive(Clone)]
 pub struct SumeragiHandle {
     public_wsv_receiver: watch::Receiver<WorldStateView>,
-    message_sender: mpsc::SyncSender<MessagePacket>,
     metrics: Metrics,
     last_update_metrics_mutex: Arc<Mutex<LastUpdateMetricsData>>,
     network: IrohaNetwork,
     kura: Arc<Kura>,
     queue: Arc<Queue>,
     _thread_handle: Arc<ThreadHandler>,
+    message_sender: mpsc::SyncSender<MessagePacket>,
```

Comment:
Does this need to be dropped last for some reason?

Reply:
Relevant to #3584

Reply:
Can you please add a comment explaining it then?